indico/modules/events/schemas.py

```diff
@@ -0,0 +1,19 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2019 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from indico.core.marshmallow import mm
+from indico.modules.events.models.principals import EventPrincipal
+from indico.util.marshmallow import PrincipalPermissionList
+
+
+class EventPermissionsSchema(mm.ModelSchema):
```

Comment:
does it have to inherit from `mm.ModelSchema` if it doesn't specify the actualy `Meta` class with `model` attribute? Can't you use `mm.Schema`?

Reply:
Good point, I've used `Meta` before and then I forgot to change that